### PR TITLE
add NCL license

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #2386

### CODE REVIEW

File: src/NCL.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+         <copyrightText>
+            <p>
+            Copyright (c) 2004 the University Corporation for
+            Atmospheric Research ("UCAR"). All rights reserved.
+            Developed by NCAR's Computational and Information
+            Systems Laboratory, UCAR, www.cisl.ucar.edu.
+            </p>
+         </copyrightText>
+         <p>
+            Redistribution and use <alt name="software" match=".*">of the Software</alt> in source and
```
Comment (jlovejoy) [inline src/NCL.xml:18]: the alt tag and match=.* here would mean "of the Software" could be replace with anything. I don't think we want that, right? The variation we have found has "of the Software" or that is simply omitted, so I think this should just be optional language? @swinslow - thoughts?

Comment (xsuchy) [reply src/NCL.xml:18]: Technically .* match to 0 or more characters. So, an empty string is a match too.

Comment (jlovejoy) [reply src/NCL.xml:18]: @goneall @swinslow - seems like this kind of think can be represented in two ways: alt tag, as @xsuchy has done, which is part of the regex standard; or with the optional tag, which is an SPDX xml special tag. I believe we have mostly done the latter for a case like this? It'd be good to be consistent. Thoughts? (and if then we can add something to the XML documentation)

Comment (xsuchy) [reply src/NCL.xml]: I changed the `alt` tag to `optional` as I have no strong preference and to ease the review.

Comment (jlovejoy) [reply src/NCL.xml]: actually, I realized this does matter b/c Optional means that this exact language can be there or be omitted. But if we use .* then any words can replace those in the alt tag. This is probably a good clarification to make in our XML documentation.

**Pull request merged**